src/cmds/create_plots.py

```diff
@@ -12,21 +11,32 @@
 from src.types.sized_bytes import bytes32
 from src.util.config import config_path_for_filename, load_config, save_config
 from src.util.default_root import DEFAULT_ROOT_PATH
+from src.util.path import mkdir
 from src.util.keychain import Keychain
-from src.util.path import mkdir, path_from_root
+from src.util.plot_tools import get_plot_filenames, stream_plot_info
 
 
 def log(to_log):
     print(to_log)
     sys.stdout.flush()
 
 
+def get_default_public_key() -> PublicKey:
+    keychain: Keychain = Keychain()
+    epk = keychain.get_first_public_key()
+    if epk is None:
+        raise RuntimeError(
+            "No keys, please run 'chia keys generate' or provide a public key with -f"
```

Comment:
I think it's actually `chia generate keys`

Reply:
RIght now it's the "chia keys" program, which is handled by commands/keys.py